websites/S/Studio.Design/presence.ts

```diff
@@ -0,0 +1,93 @@
+const presence = new Presence({
+		clientId: "1263373102746964009",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/zjBdwQ8.png",
+			startTimestamp: browsingTimestamp,
+		},
+		[privacy] = await Promise.all([presence.getSetting<boolean>("privacy")]),
+		{ pathname } = document.location;
+
+	switch (pathname) {
+		// Projects
+		case "/projects": {
+			presenceData.details = "Browsing Projects...";
+			break;
+		}
+		// Project Archive
+		case "/projects/archive": {
+			presenceData.details = "Browsing Project Archives...";
+			break;
+		}
+		default:
+			// Project Dashboard
+			if (pathname.includes("/projects/") && pathname.includes("/dashboard/")) {
+				if (privacy) presenceData.details = "Project Dashboard";
+				else {
+					// Project Name
+					presenceData.details = `${
+						document.querySelector(
+							".LayoutPdDashboard > header > div._flex._gap-8._flex-initial._flex-row.o-layout-project-dashboard-header-left._mr-auto._h-full._py-6._pl-24 > div > button > div > p"
+						).textContent
```

Comment:
```suggestion
					presenceData.details = privacy ? "Project dashboard" :  `${
						document?.title?.replace(" | Studio.Design", "")
```

Reply:
If the language setting is Japanese, the title is displayed as "Studio", so I fixed it using a regular expression.